app/assets/stylesheets/blacklight_gallery/_masonry.scss

```diff
@@ -0,0 +1,44 @@
+.view-type-masonry {
+  background-image: asset-url('icon-masonry.png') !important;
+  background-position: center;
+  background-repeat: no-repeat;
+  .view-icon-masonry {
+    padding-right: 12px;
+    height: 1.45em;
+    &:before { content: " "; }
+  }
+}
+
+#documents {
+  .masonry {
+    &.document {;
+      padding: 0;
+      margin: 0 5px 10px 0;
```

Comment:
I assume we're essentially using this margin as padding:

$padding-xs-horizontal (5px)
$padding-large-vertical (10px)